bluecode-pay/pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>com.example</groupId>
```

Comment:
Did you try to execute the following commands:
mvn -N (--non-recirsive) install - install parent module without installation of child modules (https://books.sonatype.com/mvnref-book/reference/running-sect-options.html)
mvn clean install - install all modules 
?